### ISSUE TITLE
Launch Game window does not resize contents properly

### ISSUE THREAD

**Issue by tothxa** [widelands/widelands#5631]
**Describe the bug**
The "Launch Game" window can increase the size of some parts, but not decrease them when the game window is resized.

**To Reproduce**
1. Start new single player game and select map to get to the "Launch Game" window
2. Resize the game window by dragging by the edges so that only the width or only the height changes. Try increasing then decreasing the size
3. See error

**Expected behavior**
All window elements should be resized properly for all changes

**Screenshots**
![shot0026](https://user-images.githubusercontent.com/80712255/198855931-b41729ce-0765-49d9-a5db-7586629e7d06.png)
![shot0027](https://user-images.githubusercontent.com/80712255/198855933-a8d90932-0b29-4051-8430-ea9ddc046dfa.png)
![shot0028](https://user-images.githubusercontent.com/80712255/198855934-500b2e84-2418-4931-9c5b-404a72e45864.png)
![shot0029](https://user-images.githubusercontent.com/80712255/198855936-0cc15886-604f-4c2c-ab0f-a1952d2207ea.png)
![shot0030](https://user-images.githubusercontent.com/80712255/198855937-e776ff76-a2b8-4c32-8469-c2487769e513.png)
![shot0031](https://user-images.githubusercontent.com/80712255/198855938-24c25f60-28fa-429d-8480-00efddd659cd.png)

**Version:**
 - OS: antix/Debian buster with backports
 - Widelands Version: v1.1

**Additional context**
Multiplayer launch window only has problem with vertical resize. If both width and height change (dragging by window corner), then the contents are resized properly. Also if e.g. width is decreased, when height is changed afterwards, the widths will be resized properly.
